stock_release_channel_shipment_advice/wizards/shipment_advice_planner.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="shipment_advice_planner_form_view">
+        <field name="model">shipment.advice.planner</field>
+        <field
+            name="inherit_id"
+            ref="shipment_advice_planner.shipment_advice_planner_form_view"
+        />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='shipment_planning_method']" position="after">
+                <field name="release_channel_id" />
+            </xpath>
```

Comment:
@sbejaoui Why do you use an xpath????
```suggestion
            <field name="shipment_planning_method" position="after">
                <field name="release_channel_id" />
            </field>
```

Reply:
I prefer xpath over field+position. it's easier to distinguish the added fields, but that's just a personal preference.